fabric-content-registries-v0/src/main/resources/quilt.mod.json

```diff
@@ -33,7 +33,10 @@
 			}
 		]
 	},
-	"mixin": "quilted_fabric_content_registries_v0.mixins.json",
+	"mixin": [
+		"quilted_fabric_content_registries_v0.mixins.json",
+		"quilted_fabric_content_registries_v0_qsl.mixins.json"
```

Comment:
Hm, I find the two-file approach weird; I'd go with a `quilt` subfolder; but yeah, maybe this is the more proper way of doing things